### PR TITLE
Pushed OrionLauncher module to the point of compiling

### PR DESCRIPTION
**Pull request by Crypnotic**
I'm not sure where progress lies with #14, but I've rewrote the bits of the launcher module that needed to be fixed after removing `OkHttp` and `Shuriken`. Downloading should work just fine, but I haven't had the time to test since it was a copy/paste from another project I am working on. 

I hope this helps to get the project moving forward again :)

### CODE REVIEW

**Comment by mikroskeem** [comment]
Will look into this PR today.

File: OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java [hunk 1]
```diff
@@ -27,8 +27,10 @@
 
 import org.checkerframework.checker.nullness.qual.NonNull;
 
-import java.io.IOException;
+import javax.net.ssl.HttpsURLConnection;
+import java.io.*;
```
Comment (mikroskeem) [inline OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java]: No star imports please

File: OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java [hunk 2]
```diff
@@ -91,17 +93,29 @@ void invoke() {
...
             System.out.println("Downloading Paperclip...");
-            Request request = new Request.Builder()
-                    .url(paperclipDownloadUrl)
-                    .get()
-                    .build();
-            try(Response response = client.newCall(request).execute()) {
-                if(response.code() != 200)
+
+            try {
+                HttpsURLConnection connection =
```
Comment (mikroskeem) [inline OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java]: Use HttpURLConnection instead.

File: OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java [hunk 3]
```diff
@@ -91,17 +93,29 @@ void invoke() {
...
+                        (HttpsURLConnection) paperclipDownloadUrl.openConnection();
+
+                connection.addRequestProperty("user-agent", "OrionMinecraft");
```
Comment (mikroskeem) [inline OrionLauncher/src/main/java/eu/mikroskeem/orion/launcher/PaperclipManager.java]: Minor: `User-Agent`

**Review by mikroskeem — CHANGES_REQUESTED** [review]

**Comment by Crypnotic** [comment]
Requested changes made

**Review by mikroskeem — APPROVED** [review]
LGTM for now.

**Comment by mikroskeem** [comment]
Regarding the progress, readme has Discord link where we can discuss about ideas.

**Pull request merged**